packages/victory-core/src/victory-util/hooks/use-animation-state.js

```diff
@@ -60,7 +60,7 @@ export const useAnimationState = (initialState = INITIAL_STATE) => {
   // This is a copy of Wrapper.setAnimationState
   const setAnimationState = React.useCallback(
     (props, nextProps) => {
-      if (!props.animate) {
+      if (!props || !props.animate) {
```

Comment:
Can we use optional chaining here too?

Reply:
You could also just destructure the props you need: `const {animate, children} = props;` and then check for those.